### ISSUE TITLE
Removal of <defs> breaks SVG images

### ISSUE THREAD

**Issue by rondonjon** [sairion/svg-inline-loader#12]
I have noticed the recent addition of:

```
+    // Non-displayed elements
+    [/<title>.*<\/title>/gi, ""],
+    [/<desc>.*<\/desc>/gi, ""],
+    [/<defs>.*<\/defs>/gi, ""],
```

Although it's true that these elements are never "displayed" directly, SVG images are likely to break if `<defs>` is removed, because this is where gradients, clip paths etc. are often stored.

IMO at least `<defs>` should be removed or replaced with a RegExp that only removes the `<defs>` if they're empty or contain only whitespace.

**Comment by sairion** [comment]
I see. The main reason I have done it is Photoshop SVG exports contains CSS class declaration which applied as global, if inlined. Actually, those regexps never worked (should be removed) and https://github.com/sairion/svg-inline-loader/blob/master/index.js#L26-L31 and https://github.com/sairion/svg-inline-loader/blob/master/index.js#L60-L62 is the part which removes it. 
I will make it optional, turned off as default.

**Comment by sairion** [comment]
This is fixed in `@0.3.0` release and tag removal is optional now.
